libraries/Microsoft.Bot.Builder.Expressions/BuiltInFunctions.cs

```diff
@@ -633,6 +601,10 @@ private static (object value, string error) ExtractElement(Expression expression
                             error = $"{instance} is not a collection.";
```

Comment:
This is not correct, now it can be either an object or collection.

Reply:
Fix it in the next commit